tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -385,83 +480,56 @@ void test_sphere_boundaries_equiangular() {
           "an outflow-type boundary condition, you must use that."));
 }
 
-void test_sphere_factory_equiangular() {
-  INFO("Sphere factory equiangular");
-  for (bool use_boundary_conditions : {false, true}) {
-    const auto sphere = make_domain_creator(
-        "Sphere:\n"
-        "  InnerRadius: 1\n"
-        "  OuterRadius: 3\n"
-        "  InnerCubeSphericity: 0.0\n"
-        "  InitialRefinement: 2\n"
-        "  InitialGridPoints: [2,3]\n"
-        "  UseEquiangularMap: true\n"
-        "  TimeDependence: None\n",
-        use_boundary_conditions);
-    const double inner_radius = 1.0;
-    const double outer_radius = 3.0;
-    const size_t refinement_level = 2;
-    const std::array<size_t, 2> grid_points_r_angular{{2, 3}};
-    test_sphere_construction(
-        dynamic_cast<const creators::Sphere&>(*sphere), inner_radius,
-        outer_radius, 0.0, true, grid_points_r_angular,
-        {7, make_array<3>(refinement_level)}, use_boundary_conditions);
-  }
-}
-
-void test_sphere_boundaries_equidistant() {
-  INFO("Sphere boundaries equidistant");
+void test_sphere_boundaries() {
+  INFO("Ensure sphere boundaries are equidistant");
   const double inner_radius = 1.0;
   const double outer_radius = 2.0;
-  const size_t refinement = 2;
-  const std::array<size_t, 2> grid_points_r_angular{{4, 4}};
+  const std::array<size_t, 2> refinement_array{{3}};
```

Comment:
size 2 but initialized with 1 element? Does this compile?

Reply:
This doesn't need to be an array; I just made it a `size_t`